pkg/volume/gcepd/attacher_test.go

```diff
@@ -300,113 +474,128 @@ type attachCall struct {
 	nodeName types.NodeName
 	readOnly bool
 	regional bool
-	ret      error
+	retErr   error
 }
 
 type detachCall struct {
 	devicePath string
 	nodeName   types.NodeName
-	ret        error
+	retErr     error
 }
 
 type diskIsAttachedCall struct {
-	diskName   string
-	nodeName   types.NodeName
-	isAttached bool
-	ret        error
+	attachedDisks disksAttachedMap
```

Comment:
diskIsAttached means there is only one disk to check? What is the reason for change it to Map?

Reply:
I think I understand the reason. But the name does not match well

Reply:
I personally think the naming here is fine and DiskIsAttachedCall still adequately describes the purpose of that struct, which is to define how the DiskIsAttached function should be executed. The only thing I changed is turning it from supporting a single disk to multiple disks over multiple calls.